packages/components/organisms/f-cookie-banner-static/package.json

```diff
@@ -11,7 +11,7 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-cookie-banner": "3.9.0"
+    "@justeat/f-cookie-banner": "4.0.0"
```

Comment:
Don't think this is intended, Jamie has a PR for his currently anyway - https://github.com/justeat/fozzie-components/pull/2006